_opcodes/op135.html

```diff
@@ -17,3 +17,9 @@
 <br/>
 <br/>This effect sets stat #59 (POLYMORPHED) to 1 but only if param #2 is 0.<br/>
 If the resource key is empty, other polymorph effects are removed, allowing for removal of permanent effects.
+
+{% capture note %}
+At least in EEs, when param #2 is 0, this opcode sets Strength, Dexterity, Resistances (excluding Poison and Magic Damage), Base AC, Specific AC (Crush/Slash/Pierce/Missile), and Base APR.
```

Comment:
Universally true. IWD2 doesn't change STREXTRA, but it has its own opcode description. Of course this opcode also changes the animation id.